dali/kernels/slice/slice_gpu.cuh

```diff
@@ -72,6 +72,9 @@ struct SliceBlockDesc {
   uint64_t size;
 };
 
+template<typename OutputType>
+constexpr int coalesced_pixels = sizeof(OutputType) >= 4 ? 1 : 4 / sizeof(OutputType);
```

Comment:
Nitpick: perhaps it should be "values", not "pixels". An RGB pixel consists of 3 values and I don't think this is recognized anyhow.